helpers/driverLicenseFields.js

```diff
@@ -0,0 +1,89 @@
+const driverLicenseFields = {
```

Comment:
Ideally this would be brought in via a separate module -- and we'd just return raw data from this scanner that could be parsed externally. I'll file an issue. It's fine for now.

Reply:
https://github.com/digitalbazaar/bedrock-vue-pdf417/issues/9